certif/app/styles/pages/authenticated/sessions/no-session-panel.scss

```diff
@@ -1,17 +1,20 @@
 .no-session-panel {
   display: flex;
   flex-direction: column;
+  align-items: center;
```

Comment:
**Proposition de css :** Je propose de revoir les attributs coté css (pour nettoyer un peu)

classe .no-session-panel, n'avoir plus que : 
```
padding: 92px 0 100px 0;
display: flex;
align-items: center;
```

.no-session-panel__link-to-create :

```
display: flex;
flex-direction: column;
```

Définir une taille à l'image, car elle dépend de son parent. D'après figma sa taille est de : 293x327

Je propose donc de lui créer une classe et d'y mettre : 
```
max-width : 293px;
```